src/aura/AuraAvatarMultiToken.sol

```diff
@@ -532,18 +532,24 @@ contract AuraAvatarMultiToken is BaseAvatar, PausableUpgradeable, AuraAvatarUtil
     /// @return totalBal_ Pending BAL rewards.
     /// @return totalAura_ Pending AURA rewards.
     function pendingRewards() public view returns (uint256 totalBal_, uint256 totalAura_) {
-        uint256 balEarned;
-        uint256 length = baseRewardPools.length();
+        totalBal_ = BAL.balanceOf(address(this));
+        totalAura_ = AURA.balanceOf(address(this));
 
+        uint256 length = baseRewardPools.length();
         for (uint256 i; i < length;) {
-            balEarned += IBaseRewardPool(baseRewardPools.at(i)).earned(address(this));
+            IBaseRewardPool baseRewardPool = IBaseRewardPool(baseRewardPools.at(i));
+
+            uint256 balEarned = baseRewardPool.earned(address(this));
+            uint256 balEarnedAdjusted = (balEarned * AURA_BOOSTER.getRewardMultipliers(address(baseRewardPool)))
```

Comment:
confirmed the addition of new `reward multiplier` within the aura migration infra, which can limit the amount aura minted per `reward contract`, defined on the boster,  instead of the minting aura based on the 100% of bal rewards earned ✅ 

e.g: https://etherscan.io/address/0x4EFc8DED860Bc472FA8d938dc3fD4946Bc1A0a18#code#F32#L724